test/e2e/play_build_test.go

```diff
@@ -84,17 +83,17 @@ LABEL marge=mom
 		Expect(os.Chdir(yamlDir)).To(Succeed())
 		defer func() { (Expect(os.Chdir(cwd)).To(BeNil())) }()
 
-		session := podmanTest.Podman([]string{"play", "kube", "top.yaml"})
+		session := podmanTest.Podman([]string{"kube", "play", "top.yaml"})
```

Comment:
I didn't check but please make sure that there are still some caller of `play kube` to make sure the alias continues to work.

Reply:
Only a hundred and sixty-three instances.